docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[synonym-management-apis]]
+== Synonym Management APIs
+
+beta::[]
+
+++++
+<titleabbrev>Synonyms Management APIs</titleabbrev>
+++++
+
+---
+
+Synonyms Management APIs let you create and manage synonyms sets.
+
+Synonyms sets can be used to configure <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>>.
+
+Once a synonyms set is used in a synonym token filter, these token filters can be used as part of a <<search-analyzer,search analyzers>>.
```

Comment:
can be used => must be used or "can only be used"

Synonym token filter can be used  only in search analyzers ...

Reply:
I'm rephrasing this as "are used as part", to leave the "can" out 👍